modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteServerImpl.java

```diff
@@ -281,9 +386,11 @@ private static void ackBanner() {
 
     private static void sync(CompletableFuture<Void> future) {
         try {
-            future.join();
-        } catch (CompletionException e) {
+            future.get();
```

Comment:
Can you explain the change?

Reply:
`join()` is uninterruptible, hence IMO we should avoid it